packages/extension/src/hosted/api/vscode/ext.host.tests.ts

```
@@ -263,6 +281,14 @@ export class ExtHostTestsImpl implements IExtHostTests {
       get resolveHandler() {
         return collection.resolveHandler;
       },
+      invalidateTestResults: (items) => {
+        if (items === undefined) {
+          this.proxy.$markTestRetired(undefined);
+        } else {
+          const itemsArr = items instanceof Array ? items : [items];
```

Biome

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)